lib/resources/example_json/r4_examples_questionnaire_response.rb

```diff
@@ -4,7 +4,33 @@ module Cerner
   module Resources
     R4_QUESTIONNAIRE_RESPONSE_ENTRY ||= {
       'resourceType': 'QuestionnaireResponse',
-      'id': 'SH-12508041'
+      'id': 'SH-12508041',
+      'questionnaire': 'https://fhir-ehr-code.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/Questionnaire/SH-12508041',
+      'status': 'in-progress',
+      'subject': {
+        'reference': 'Patient/12508041'
+      },
+      'encounter': {
+        'reference': 'Encounter/36209678'
+      },
+      'authored': '2019-10-25T14:57:43Z',
+      'item': [
+        {
+          'linkId': '93',
+          'text': 'Tobacco',
+          'item': [
+            {
+              'linkId': '93-123',
+              'text': 'Number of years:',
+              'answer': [
+                {
+                  'valueQuantity': '20'
+                }
+              ]
+            }
+          ]
+        }
+      ]
```

Comment:
text field?
And lets show an example of each type of answer string, coding, quantity

Reply:
I thought about that, but according to business validation we can't have multiple answers with one being type quantity.

Reply:
6143000a142973a554ec7ff16230c4e321f47768

Reply:
the one item with the coding answer will need the choice answer extension, and also lets add a value string to it to show how to add the other text field

Reply:
9f140942a1b3bec2eec375f79b9a284358c7d0cc

Reply:
third one's structure is wrong, should be like:
```
                "answer": [
                    {
                         "valueString": "other text answer"
                    },
                    {
                        "valueCoding": { 
                            "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/nomenclature",
                            "code": "10985140"
                        }
                    }
]
```

Reply:
whoops, fixed: 3c70eb3e8f903fb3e8aa8f3c2008e285b1d9705b